config/sync/itk_pretix.pretixconfig.yml

```diff
@@ -0,0 +1,5 @@
+pretix_url: 'https://pretix.eu/'
```

Comment:
This file should not be committed! The config should be ignored by the `config_ignore` module.